server/src/main/java/org/opensearch/cluster/coordination/Coordinator.java

```diff
@@ -1693,6 +1704,7 @@ public void onSuccess(String source) {
                                     updateMaxTermSeen(getCurrentTerm());
 
                                     if (mode == Mode.LEADER) {
+                                        closePrevotingAndElectionScheduler();
```

Comment:
now lets say election doesn't succeed after x attempts, then election scheduler will stop scheduling at all. Are those attempts unbounded or finite?

Reply:
Election attempts are unbounded. They will keep scheduling after the timeout configured.